src/library_webgl.js

```diff
@@ -2421,17 +2421,18 @@ for (/**@suppress{duplicate}*/var i = 0; i < {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
     GL.validateGLObjectID(GLctx.currentProgram.uniformLocsById, location, 'glUniform1iv', 'location');
     assert((value & 3) == 0, 'Pointer to integer data passed to glUniform1iv must be aligned to four bytes!');
 #endif
+    if (!count) return;
```

Comment:
The original `count &&` was added to save on code size, I wouldn't like to lose that, it doesn't seem necessary?